api/src/controllers/member.controller.ts

```diff
@@ -29,6 +29,25 @@ class MemberController {
    */
   constructor(private memberService: MemberService, private homeService: HomeService) {}
 
+  public async getAdminLevel(request: Request, response: Response): Promise<object> {
+    const session = this.memberService.decryptSession(request, response);
+    if (!session) return;
+    const admin = await this.memberService.canAdmin(session.id);
+    const superAdmin = await this.memberService.canSuperAdmin(session.id);
+    response.status(200).json({admin, superAdmin});
```

Comment:
Sending a set of booleans to represent a user level seems like it will eventually lead us to doing a whole bunch of conditional checks in other places. It'd be better if we could just return an enum that represents the user's role i.e. `UserRole.Admin`, `UserRole.SuperAdmin`. It reduces the question down to one data point, rather than N.

Reply:
Will relook at this.